test/Interop/Cxx/class/conforms-to-errors.swift

```diff
@@ -16,6 +16,58 @@ struct  __attribute__((swift_attr("conforms_to:SwiftTest.X"))) CX {};
 struct  __attribute__((swift_attr("conforms_to:SwiftTest.A"))) CA {};
 struct  __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CB {};
 
+struct  __attribute__((swift_attr("conforms_to:X")))
+    __attribute__((swift_attr("conforms_to:X"))) CXX {};
+struct  __attribute__((swift_attr("conforms_to:X")))
+    __attribute__((swift_attr("conforms_to:Mod.X"))) CXModX {};
+struct  __attribute__((swift_attr("conforms_to:X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.X"))) CXTextX {};
+struct  __attribute__((swift_attr("conforms_to:X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.A"))) CXA {};
+struct  __attribute__((swift_attr("conforms_to:X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CXB {};
+struct  __attribute__((swift_attr("conforms_to:X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.C"))) CXC {};
+
+
+struct  __attribute__((swift_attr("conforms_to:Mod.X")))
+    __attribute__((swift_attr("conforms_to:Mod.X"))) CModXModX {};
+struct  __attribute__((swift_attr("conforms_to:Mod.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.X"))) CModXTestX {};
+struct  __attribute__((swift_attr("conforms_to:Mod.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.A"))) CModXA {};
+struct  __attribute__((swift_attr("conforms_to:Mod.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CModXB {};
+struct  __attribute__((swift_attr("conforms_to:Mod.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.C"))) CModXC {};
+
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.X"))) CTestXTextX {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.A"))) CTextXA {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CTextXB {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.X")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.C"))) CTextXC {};
+
+
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.A")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.A"))) CAA {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.A")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CAB {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.A")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.C"))) CAC {};
+
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.B")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CBB {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.B")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.C"))) CBC {};
+
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.C")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.C"))) CCC {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.C")))
+    __attribute__((swift_attr("conforms_to:SwiftTest.D"))) CCD {};
```

Comment:
Is `CCD` supposed to be an error in some way that I'm missing? And if not, why include it in this test?